django_netjsongraph/templates/admin/django_netjsongraph/visualize.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n admin_urls %}
+{% load static %}
+
+{% block content %}
+<div>
+    <a class="close" href="#">&times; {% trans 'Close' %}</a>
+</div>
+<div id="legend">
+    <p><span class="link up">&nbsp;</span>link up</p>
+    <p><span class="link down">&nbsp;</span>link down</p>
+</div>
+{% block script %}
```

Comment:
we are duplicating information here, try to convert this part in a fragment which can be shared with the frontend template (and included with `{% include '../netjsongraph/netjsongraph-script.html' %}`) and overridden [in accordance with the overriding templates instructions in the README](https://github.com/netjson/django-netjsongraph#overriding-visualizer-templates)